src/run_environment/machine.ml

```diff
@@ -122,6 +122,11 @@ module Make_fun = struct
                                   ["#WIP"]). *)
       | `Custom of string (** Pass arbitrary data (useful for temporary
                               extensions/experiements outside of Biokepi). *)
+      | `Self_identification of string list
```

Comment:
This might be nicer to match with downstream as just `Self_identification of string` instead. 

We could also use a type instead. If we used a type instead of a string, Merlin/the compiler could help us figure out what we can match on. As it is, someone will have to go through the source code to figure out what they can match on (and it could change/break on them in the future without a warning).

(Still not sure, even with the Hamkepi PR, that this solution is as simple for the user as passing `?requirements` down from the tool itself…)